linode/networkreservedip/framework_models.go

```diff
@@ -0,0 +1,47 @@
+package networkreservedip
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/linodego"
+
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+)
+
+type ReservedIPModel struct {
```

Comment:
Are we missing the ability to filter by fields? (If so, can be a follow up PR, probably)

Reply:
Filtering on region makes sense. I will be implementing the filter ability now.